### PR TITLE
Make `isStressTest` and the related properties top-level

### PR DESCRIPTION
**Pull request by ndkoval**

### CODE REVIEW

File: kotlinx-coroutines-core/common/test/TestBase.common.kt [hunk 1]
```diff
@@ -10,10 +10,10 @@ import kotlinx.coroutines.flow.*
 import kotlin.coroutines.*
 import kotlin.test.*
 
-public expect open class TestBase constructor() {
-    public val isStressTest: Boolean
-    public val stressTestMultiplier: Int
+public expect val isStressTest: Boolean
```
Comment (qwwdfsad) [inline kotlinx-coroutines-core/common/test/TestBase.common.kt]: What's the rationale behind this change?

Comment (elizarov) [reply kotlinx-coroutines-core/common/test/TestBase.common.kt]: It is needed in two places:
* For lincheck tests (which do not extend TestBase)
* To tweak a list of parameters to parameterized tests, which has to happen in a static context using `isStressTest`.
